File: pinyougou-sellgoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();

    public PageResult findpage(int pageNum,int pageSize);

    public void add(TbBrand tbBrand);

    public TbBrand findOne(Long id);

    public void update(TbBrand tbBrand);

    public void delete(Long[] ids);

    public PageResult search(TbBrand tbBrand,int pageNum,int pageSize);

    public List<Map> selectOptionList();

}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
//品牌服务
app.service('brandService',function ($http) {
    this.findAll = function () {
        return $http.get('/brand/findAll.do');
    };

    this.findPage = function (pageNum,pageSize) {
        return	$http.get('/brand/findPage.do?pageNum='+pageNum+'&pageSize='+pageSize);
    };

    this.findOne = function (id) {
        return $http.get('/brand/findOne.do?id='+id);
    };

    this.add = function (entity) {
        return $http.post('/brand/add.do',entity);
    };

    this.update = function (entity) {
        return $http.post('/brand/update.do',entity);
    };

    this.dele = function (ids) {
        return $http.get('/brand/delete.do?ids=' + ids);
    };

    this.search = function (pageNum,pageSize,searchEntity) {
        return $http.post('/brand/search.do?pageNum='+pageNum+'&pageSize='+pageSize,searchEntity);
    };

    this.selectOptionList = function () {
        return $http.get('/brand/selectOptionList.do');
    };
});